tests/mock_tables/database_global.json

```diff
@@ -0,0 +1,26 @@
+{
+    "INCLUDES" : [
+        {
+            "include" : "database_config.json"
+        },
+
+
+        {
+            "namespace" : "asic0",
+            "include" : "./asic0/database_config.json"
+
+        },
+        {
+            "namespace" : "asic1",
+            "include" : "./asic1/database_config.json"
+
+        },
+        {
+            "namespace" : "asic2",
+            "include" : "./asic2/database_config.json"
+
+        }
+
+    ],
```

Comment:
reformat the json file?

Reply:
Fixed.

Reply:
still

---
In reply to: [427636346](https://github.com/Azure/sonic-snmpagent/pull/126#discussion_r427636346) [](ancestors = 427636346)

Reply:
Removed additional lines.